charts/team-ns/templates/ingress/harbor-public.yaml

```diff
@@ -5,7 +5,7 @@
 {{- $cm := index $v.apps "cert-manager" }}
 {{- $ingress :=  $v.ingress.platformClass }}
 {{- $name :=  printf "nginx-team-%s-platform-public-open-forward-harbor" $v.teamId }}
-{{- $secretName := ternary "otomi-byo-wildcard-cert" "otomi-cert-manager-wildcard-cert" (eq $cm.issuer "byo-wildcard-cert") }}
+{{- $tlsSecretName := ternary $cm.externallyManagedTlsSecretName "otomi-cert-manager-wildcard-cert" (or (eq $cm.issuer "byo-wildcard-cert") (eq $cm.issuer "externally-managed-tls-secret")) }}
```

Comment:
I would move this logic to derived.gotmpl file and define:
_derived.tlsSecretName and reference everywhere that it is needed

mso you do not need to repeat this logic all over.